<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XoloProjectCreator</class>
 <widget class="QWidget" name="XoloProjectCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Creator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_image">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="pcreator.qrc">:/xolo/icons/xolo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>xolo - pipeline</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="pc_label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Project Creator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="project_label">
         <property name="minimumSize">
          <size>
           <width>3</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Project Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_project">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Set the project name </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="directory_label">
         <property name="text">
          <string>Directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_directory">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_search">
         <property name="toolTip">
          <string>Search the  main directory where you want to create the project this can not be change after the creation fo the project because if you change it the project will break</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="type_label">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbb_type">
         <property name="toolTip">
          <string>Type of project  this will define the folder structure for the project </string>
         </property>
         <item>
          <property name="text">
           <string>Commercial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VFX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Animation</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shot</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="format_label">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbb_format">
         <property name="toolTip">
          <string>Resolution for the delivery / work of the project  this is need it to set all scene files with the correct resolution</string>
         </property>
         <item>
          <property name="text">
           <string>UHD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1920p</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1280p</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2kDCP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4kDCP</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fps_label">
         <property name="text">
          <string>Fps:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbb_fps">
         <property name="toolTip">
          <string>Define how many fps will be set for the project </string>
         </property>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23.976</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>60</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>120</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create">
       <property name="toolTip">
        <string>Create project and base folder structure need it for  sync  project ith the tools</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pcreator.qrc"/>
 </resources>
 <connections/>
</ui>
